generated/nidcpower/library.py

```diff
@@ -76,372 +76,372 @@ def niDCPower_Abort(self, vi):  # noqa: N802
                 self.niDCPower_Abort_cfunc = self._library.niDCPower_Abort
                 self.niDCPower_Abort_cfunc.argtypes = [ViSession]  # noqa: F405
                 self.niDCPower_Abort_cfunc.restype = ViStatus  # noqa: F405
-        return self.niDCPower_Abort_cfunc(vi).value
+        return self.niDCPower_Abort_cfunc(vi)
```

Comment:
If I understand correctly, this used to return an int. Now it returns a ViStatus/ctypes.c_long.
Is that right?